.github/workflows/demo-testing.yml

```diff
@@ -1,12 +1,18 @@
 name: Demo Testing
 
-on: ["push", "pull_request"]
+on:
```

Comment:
this was needed because we were running push on all branches including revert and cherry-picks created by automation